.github/workflows/artifact-policy-deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy Artifact Registry Policy
+
+on:
+  push:
```

Comment:
It might be better to GitHub Actions workflow to run only when changes are pushed to the `artifact-registry-policy.json` or this file on the `main` branch (that way it doesn't run the script when you are pushing to non-`main` files).

You could just add this

```
on:
  push:
    branches:
      - main
    paths:
      - artifact-registry-policy.json
      - .github/workflows/artifact-policy-deploy.yml
```

Reply:
sure, i'll change it!